app/full_text_search.py

```diff
@@ -51,7 +58,9 @@ def search(repo, query_txt, max_results):
         query_string=query_txt, options=options))
     for document in index_results:
         id = document.fields[0].value
-        results.append(model.Person.get_by_key_name(repo + ':' + id))
+        result = get_record_from_datastore(repo, id)
```

Comment:
Use Person.get() instead:
https://github.com/google/personfinder/blob/master/app/model.py#L209

Person.get() also has feature to check if it's expired.